vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -51,37 +50,35 @@ A customized image can include:
 * New APIs 
 * Legacy Entando plugins
 * New database tables  
-* Other extensions to the **entando-core** 
+* Other extensions to the entando-core
```

Comment:
why isn't the entando-core getting love anymore?? if we're bolding entando-de-app shouldn't its counterpart get the same treatment? (applies to above instance as well)

Reply:
backticked

Reply:
i personally avoid bolding and only use backtracking for commands or (not always) the names of software functions/objects/tools, etc.